zenodo/modules/records/data/objecttypes.json

```diff
@@ -373,6 +373,25 @@
   },
   "csl": "article"
 },
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "computational-notebook",
+  "id": "https://zenodo.org/objecttypes/publication/computationalnotebook#",
+  "title": {
+    "en": "Computational Notebook"
+  },
+  "title_plural": {
+    "en": "Computational Notebooks"
+  },
+  "datacite": {"general": "Software", "type": "Computational Notebook"},
+  "eurepo": "info:eu-repo/semantics/other",
+  "openaire": {
+    "resourceType": "0029",
+    "type": "software"
+  },
+  "schema.org": "https://schema.org/SoftwareSourceCode",
+  "csl": "article"
+},
```

Comment:
We don't want to add new types also, just update the `datacite` keys of the existing types we have over to the new values from v4.4.